package.json

```diff
@@ -8,7 +8,7 @@
   "description": "Select2 Foundation Theme",
   "dependencies": {
     "grunt-contrib-watch": "^0.6.1",
-    "grunt-sass": "^1.1.0"
+    "grunt-contrib-sass": "^1.1.0"
```

Comment:
@egemensarica I could not run npm install with `grunt-contrib-sass` at `1.1.0` but it does install at `1.0.0`. Can you change the dependency to `1.0.0`?